src/app/services/global.service.ts

```diff
@@ -321,8 +322,12 @@ export class GlobalService {
       this.showToast('error', err.error['error'], 5);
```

Comment:
Can we use this service and this ` handleApiError`  instead  [this service](https://github.com/Cloud-CV/EvalAI-ngx/pull/104/files#diff-477bb86affda2e5cdadf1244bbe6757f) ?

Reply:
If u mean whether we should this handleApiError or [this service](https://github.com/Cloud-CV/EvalAI-ngx/pull/104/files#diff-477bb86affda2e5cdadf1244bbe6757f).
then this handleApiError method would be better as it handles all the api errors. Therefore adding the logic here itself might be correct.